### PR TITLE
Updated for Qdrant VDB

### PR DESCRIPTION
**Pull request by atarora**
Updated for Qdrant VDB

### CODE REVIEW

File: docs/tools/vdb_table/data/qdrant.json [hunk 1]
```diff
@@ -73,18 +73,18 @@
...
   "embeddings_structured": {
     "support": "",
     "source_url": "",
     "comment": ""
   },
   "rag": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://colab.research.google.com/github/qdrant/examples/blob/master/rag-openai-qdrant/rag-openai-qdrant.ipynb",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/qdrant.json:87]: This column is for "Calls LLM internally for RAG". Every VDB would be able to call OpenAI as in the notebook you have shared.

File: docs/tools/vdb_table/data/qdrant.json [hunk 2]
```diff
@@ -108,14 +108,14 @@
...
   "pricing": {
-    "value": "",
-    "source_url": "",
+    "value": "Qdrant Pricing (free tier available)",
+    "source_url": "https://qdrant.tech/pricing/",
     "comment": ""
   },
   "in_process": {
-    "support": "none",
+    "support": "full",
```
Comment (dhruv-anand-aintech) [inline docs/tools/vdb_table/data/qdrant.json:116]: this should be 'partial' as it is only in a specific case

**Review by dhruv-anand-aintech — CHANGES_REQUESTED** [review]
Please address the comments

**Comment by atarora** [comment]
> Please address the comments

Done

**Review by dhruv-anand-aintech — APPROVED** [review]

**Comment by dhruv-anand-aintech** [comment]
Thanks for keeping the table up-to-date, @atarora!

**Pull request merged**